packages/host/app/components/mode-switcher.gts

```diff
@@ -0,0 +1,138 @@
+import Component from '@glimmer/component';
+import { BoxelDropdown, Button, Menu } from '@cardstack/boxel-ui';
+import { svgJar } from '@cardstack/boxel-ui/helpers/svg-jar';
+import { menuItemFunc, MenuItem } from '@cardstack/boxel-ui/helpers/menu-item';
+import { fn } from '@ember/helper';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import map from 'ember-composable-helpers/helpers/map';
+import { service } from '@ember/service';
+import RouterService from '@ember/routing/router-service';
+import type OperatorModeStateService from '../services/operator-mode-state-service';
+import type CardService from '../services/card-service';
+import { on } from '@ember/modifier';
+
+interface Signature {
+  Element: HTMLElement;
+}
+
+type Mode = {
+  icon: string,
+  label: string,
+}
+
+export default class ModeSwitcher extends Component<Signature> {
```

Comment:
Let's name this file and class `SubmodeSwitcher` / `submode-switcher.gts`

Reply:
updated [27c17c8](https://github.com/cardstack/boxel/pull/554/commits/27c17c872d5dbd817ef579e0fd7ed2123b3f639b)